pkg/remotewrite/remotewrite.go

```diff
@@ -39,19 +38,18 @@ func New(params output.Params) (*Output, error) {
 		return nil, err
 	}
 
-	remoteConfig, err := config.ConstructRemoteConfig()
+	clientConfig, err := config.RemoteConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	// name is used to differentiate clients in metrics
-	client, err := remote.NewWriteClient("xk6-prwo", remoteConfig)
+	wc, err := remote.NewWriteClient(config.URL.String, clientConfig)
 	if err != nil {
 		return nil, err
```

Comment:
Maybe wrap an error here, to make it more clear to the end user, something like: `failed to initialize Prometheus remote write` or something? WDYT?

Reply:
Added in the latest fixup